app/routes/refund-policy/refund-policy.module.scss

```diff
@@ -0,0 +1,58 @@
+.root {
+    margin: 60px 0px;
+}
+
+.title {
+    font: var(--heading2);
+    margin-bottom: 40px;
+    text-align: center;
+}
+
+.section {
+    width: 65%;
+    margin: 0 auto 40px;
+}
+
+.subtitle {
+    font-size: 22px;
+    margin-bottom: 8px;
+    text-align: center;
+}
+
+.text {
+    text-align: center;
+}
+
+@media screen and (max-width: 1024px) {
```

Comment:
Let's use breakpoint variables. They are available now from the `common.scss` file.